### DISCUSSION TITLE
Very low resolution textures are blurred

### DISCUSSION THREAD

**Discussion by TheNoswad** [google/model-viewer#3094] [Q&A]
Hi.

I'm trying to texture a model with a pixel art texture, and the texture is blurry. I would rather see the pixels instead of blurring them together. Here is a screenshot.
![image](https://user-images.githubusercontent.com/64293615/147862420-5bf25473-13e6-46cd-b167-84ac4cad8606.png)

Usually this problem can be fixed by enabling nearest neighbor or something similar, but I didn't see anything in the documentation or examples.

**Comment by elalish** [comment]
That's part of the glTF spec; you can specify nearest neighbor interpolation in your `sampler` and we'll respect it. If you can't  find an  exporter that'll do it for you, you can edit it by hand pretty easily with the glTF VSCode extension.

**Reply by elalish** [reply]
You may also want to consider the `materials_unlit` extension if you're not trying to  make a physical model.
